template-theory/index.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+    <link rel="stylesheet" href="./css/grib.css">
+    <link rel="stylesheet" href="./css/form-control.css">
+    <link rel="stylesheet" href="./css/lagre-screen.css">
+    <link rel="stylesheet" href="./css/medium-screen.css">
+    <link rel="stylesheet" href="./css/small-screen.css">
+    <title>Theory - Template </title>
+</head>
+
+<body>
+    <div class="header">
+        <div class="menu">
+            <a href="/" class="logo">Theory</a>
+            <ul class="nav">
+                <li>
+                    <a href="#">Home</a>
+                </li>
+                <li>
+                    <a href="#">Generic</a>
+                </li>
+                <li>
+                    <a href="#">Elements</a>
+                </li>
+            </ul>
+            <a href="#" onclick="openMenuMobile(true)" class="navPanelToggle"><span class="fa fa-bars"></span></a>
+            <div id="nav-mobile">
```

Comment:
```suggestion
            </ul>

            <a href="#" onclick="openMenuMobile(true)" class="navPanelToggle"><span class="fa fa-bars"></span></a>

            <div id="nav-mobile">
```
Tách các khối ra cho dễ nhìn